app/Models/ResetPassword.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\DB;
+
+class ResetPassword extends Model
+{
+    use HasFactory;
+
+    public function addResetToken($email, $token)
+    {
+        DB::table('password_reset_tokens')->insert([
+            ['email' => $email, 'token' => $token, 'created_at' => now()]
+        ]);
+    }
+
+    public function findToken($data)
+    {
+       return DB::table('password_reset_tokens')->where('token', $data)->first();
+    }
+
+    public function deleteToken($data)
```

Comment:
useless method, you can use 'delete' method on the entity. ex.: $token->delete()